### PR TITLE
Update cluster ssh

### PR DESCRIPTION
**Pull request by carolineechen**
something broke with the ssh flow of a local (docker) ondemand cluster that isn't saved, with loading the ssh proxy command from a generated sky config. 

can just directly call `ssh <cluster_name>`, sky sets up the ssh config already and it should flow. no longer running into the previous need to call `conda deactivate` for docker images either

### CODE REVIEW

**Comment by carolineechen** [comment]
* **#1597** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1597?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1597?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: runhouse/main.py [hunk 1]
```diff
@@ -155,46 +154,21 @@ def cluster_ssh(cluster_name: str):
...
-        import sky
+        try:
+            import sky
 
-        from runhouse import OnDemandCluster
-        from runhouse.constants import DEFAULT_SSH_PORT
-        from runhouse.resources.hardware.utils import _run_ssh_command
-
-        state = sky.status(cluster_names=[cluster_name], refresh=False)
+            state = sky.status(cluster_names=[cluster_name], refresh=False)
```
Comment (jlewitt1) [inline runhouse/main.py:160]: i guess it was already like this before, but running sky status here is really only relevant if the cluster's launcher is set to local no?

Comment (jlewitt1) [reply runhouse/main.py:160]: oh i see, this is if we fail to even load it. ya i guess may as well try checking with local sky

Comment (carolineechen) [reply runhouse/main.py:160]: yea, if the launcher is den the cluster should be saved/loaded down and run through the first part? but can also make it explicit here

**Review by jlewitt1 — APPROVED** [review]

File: runhouse/main.py (hunk 1 above)
Comment (jlewitt1) [reply runhouse/main.py:160]: i guess most explicit would be to exit if a value error is raised and launcher is set to den

Comment (carolineechen) [reply runhouse/main.py:160]: actually if we can't load a cluster, we can't determine for sure if it was supposed to be launched with den or local, so I think leaving it probably handles the cases already (theoretically will be saved/loaded. and if for some reason it isn't, it won't be found locally if launched through den, and will raise an error then)

**Pull request merged**